src/contract-details/ContractDetailsCard.js

```diff
@@ -18,54 +19,28 @@ export default class ContractDetailsCard extends Component {
 		const workAroundForBackendBugIsDigitTrade = contract.shortcode && contract.shortcode.includes('DIGIT');
 		const validToSell = contract.is_valid_to_sell === 1 && !contract.is_expired;
 		const validationError = contract.validation_error;
-		const potentialProfit = contract.payout - contract.buy_price;
-		const profit = sold && (contract.sell_price - contract.buy_price);
+		const potentialProfit = (contract.payout - contract.buy_price).toFixed(2);
```

Comment:
From my more than 10 years of experience in using money, I've yet encounter money using more than 2 decimal places 

=P

But for the sake of clarity, I'll add the method to utils and use it here